src/router.ts

```diff
@@ -14,13 +14,13 @@ const authMiddleware = expressjwt({
 });
 
 // Support for email+password
-router.post('/signin', (req, res) => signinController(req, res));
-router.post('/signup', (req, res) => signupController(req, res));
+router.post('/apollo-signin', (req, res) => signinController(req, res));
```

Comment:
@jupiter007 you may need to update these in the frontend

Reply:
Thanks @briri . I created this PR to go with this change: https://github.com/CDLUC3/dmsp_frontend_prototype/pull/100